### DISCUSSION TITLE
Can we forward multiple blocks quickly?

### DISCUSSION THREAD

**Discussion by mookim-eth** [ConsenSys-archive/ganache#3849] [General]
`evm_mine` can only forward one block, and consecutively requesting `evm_mine` for thousand times request a lot of time

Can we add another RPC method for forwarding multiple blocks quickly, like `evm_increaseblocks(10000)`?

I just want to set block height to a bigger value, no need to query these generated/skipped blocks.

**Comment by davidmurdoch** [comment]
You can. Use `evm_mine` with params `[{ blocks: "0x1234" }]`.
